autogen/oai/client.py

```diff
@@ -161,6 +169,14 @@ def create(self, params: Dict[str, Any]) -> ChatCompletion:
 
         completions: Completions = self._oai_client.chat.completions if "messages" in params else self._oai_client.completions  # type: ignore [attr-defined]
         # If streaming is enabled and has messages, then iterate over the chunks of the response.
+
+        _is_vision = is_vision_model(params.get("model", None))
+        _is_vision |= params.get("vision_model", False)
```

Comment:
Some nontrivial assumptions are made here. They need to be documented in docstr.

Reply:
Added docstring, comments, notebook.

The documentation is also updated in the website /docs/llm_configuration

Reply:
won't the field "vision_model" fail the `.create()` call as an unrecognized kwarg? Shall we use `pop` here?